numpy/core/numeric.py

```diff
@@ -1654,7 +1662,7 @@ def indices(dimensions, dtype=int):
     array([[0, 1, 2],
            [0, 1, 2]])
 
-    The indices can be used as an index into an array.
+    The indices can be used as an index into an array, if sparse is False.
```

Comment:
No need for this change. They'll work as indices either way.

Reply:
Removed it.